controllers/discovery/searchDomain.js

```diff
@@ -15,7 +15,7 @@ const SearchDomain = async (req, res) => {
 
   try {
     const domains = await sequelize.query(
-      `SELECT 
+      `SELECT distinct
         "Domain"."domain_page_id" AS "domain_id_followed",
         "Domain"."domain_name",
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `distinct` is a good way to avoid duplicate results in your query. However, it's important to note that `distinct` operates on the entire row. This means if there are other columns in your select statement not shown here, and those columns have different values for the same domain, you will still get multiple rows for the same domain. If you want to ensure uniqueness based on certain columns only (like "Domain"."domain_name"), you might need to adjust your query accordingly.
```diff
- SELECT distinct
+ SELECT
  "Domain"."domain_page_id" AS "domain_id_followed",
  "Domain"."domain_name",
  "Domain"."logo",
```
And then use `GROUP BY "Domain"."domain_name"` at the end of your query.


<!-- This is an auto-generated comment by OSS CodeRabbit -->